libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/MapboxNativeNavigator.kt

```diff
@@ -131,6 +133,22 @@ interface MapboxNativeNavigator {
      */
     fun getHistoryRecorderHandle(): HistoryRecorderHandle?
 
+    /**
+     * Returns the native class that allows the sdk to record native history files.
+     *
+     * @return null when there is no directory to write files,
+     *     or when the handle did not [HistoryRecorderHandle.build]
+     */
+    fun getCopilotHistoryRecorderHandle(): HistoryRecorderHandle?
+
+    /**
+     * Returns the native class that allows the sdk to record native history files.
+     *
+     * @return null when there is no directory to write files,
+     *     or when the handle did not [HistoryRecorderHandle.build]
+     */
+    fun getCompositeHistoryRecorderHandle(): HistoryRecorderHandle?
```

Comment:
For me "composite" handle is something that has a list or a chain of inner handles whose methods are invoked when the composite's corresponding method is invoked. That's why it's not very clear to me how you can create a "composite" handle without any other handles passed as arguments.
UPD: I see that it's working exactly like this. Then my question is: do we need a composite handle in `MapboxNavigation` at all? It's only used when creating `RouterInterface`. Can't we just pass the list of handles in the method and create the composite there? I think it would be less confusing because now we have 3 handles across multiple classes and it's not obvious which one should be used in which situation.